### PR TITLE
removing index on remove_index_models_on_record_type_and_record_id and adding new index

### PR DESCRIPTION
**Pull request by HemaHG**
Addressing the review comments: https://github.com/3scale/zync/pull/444#discussion_r622898857

### CODE REVIEW

File: db/migrate/20210504152609_add_index_models_on_record.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class AddIndexModelsOnRecord < ActiveRecord::Migration[6.1]
+  def change
+    create_table :models, if_not_exists: true do |t|
```
Comment (hallelujah) [inline db/migrate/20210504152609_add_index_models_on_record.rb:3]: You do not want to recreate the table

**Review by hallelujah — CHANGES_REQUESTED** [review]

File: db/migrate/20210504151301_remove_index_models_on_record_type_and_record_id.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class RemoveIndexModelsOnRecordTypeAndRecordId < ActiveRecord::Migration[6.1]
+  def change
+    remove_index :models, name: "index_models_on_record_type_and_record_id" , if_exists: true
+  end
+end
```
Comment (hallelujah) [inline db/migrate/20210504151301_remove_index_models_on_record_type_and_record_id.rb:5]: You can merge both migrations into one

File: db/migrate/20210504152609_add_index_models_on_record.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class AddIndexModelsOnRecord < ActiveRecord::Migration[6.1]
+  def change
+    add_index :models, [:record_id, :record_type], name: "index_models_on_record", if_not_exists: true
```
Comment (hallelujah) [inline db/migrate/20210504152609_add_index_models_on_record.rb:3]: The uniqueness constraints is gone :(

**Review by hallelujah — CHANGES_REQUESTED** [review]

File: db/migrate/20210504151301_remove_index_models_on_record_type_and_record_id.rb (hunk 2 above)
Comment (HemaHG) [reply db/migrate/20210504151301_remove_index_models_on_record_type_and_record_id.rb:5]: I thought the same but was not sure. So created two migrations. will merge both :)

**Review by hallelujah — APPROVED** [review]

**Pull request merged**